protos/flyteidl/plugins/ray.proto

```diff
@@ -19,13 +21,19 @@ message RayCluster {
   HeadGroupSpec head_group_spec = 1;
   // WorkerGroupSpecs are the specs for the worker pods
   repeated WorkerGroupSpec worker_group_spec = 2;
+  // Namespace used to create the ray cluster
+  string namespace = 3;
+  // Kubernetes service account used by the ray cluster
+  string k8s_service_account = 4;
```

Comment:
I just remembered that we could override the service account by using [pod template](https://docs.flyte.org/en/latest/deployment/configuration/general.html#compile-time-podtemplates). That said, we can remove `k8s_service_account` in the idl.

Reply:
@pingsutw how does it look like in the flytekit UX? we added `k8s_service_account` to the `RayJobConfig`. Would we not need it anymore and instead expect the user to override it using pod template?

Reply:
I think it may not be nice user experience if the user has to specify some attributes to override in `RayJobConfig` and some through a pod template

Reply:
We want to make it general because others may want to override the service account for spark, Dask, TensorFlow tasks, etc. How about adding a namespace arg to the task decorator?
```python
@task(rayJobConfig(...), service_account="")
def ray_task()
```

Reply:
Are you suggesting just keeping the Ray-specific parameters in the RayJobConfig? Are we then going to create a parameter in the task decorator for each non-Ray configuration we want to override?

Reply:
Namespace is not a common config that people will override so we can keep it in the rayjob. 
> Are you suggesting just keeping the Ray-specific parameters in the RayJobConfig? Are we then going to create a parameter in the task decorator for each non-Ray configuration we want to override?

Yes, ideally. people may want to override the service account for the spark job. any other configs you want to override?

Reply:
@pingsutw sorry we were away for a work event last week. So if I understand correctly, you're just suggesting moving `k8s_sa` outside of `rayjob` and allow users to configure it at the decorator level?

```python
@task(rayJobConfig(...), service_account="")
def ray_task():
    ...
```

Reply:
I understand the concern around generalizability (other tasks also configuring `service_account`). Is this still not a better user-experience to configure everything in the task config object versus a decorator-level argument?

```python
@task(rayJobConfig(namespace="", k8s="...",...))
def ray_task()

@task(sparkJobConfig(k8s="...",...))
def spark_task()
```